cloud/docker_client.go

```diff
@@ -469,7 +476,6 @@ func (c *dockerClientImpl) ListContainers(ctx context.Context, h *host.Host) ([]
 	containers, err := dockerClient.ContainerList(ctx, opts)
 	if err != nil {
 		err = errors.Wrap(err, "Docker list API call failed")
-		grip.Error(err)
 		return nil, err
```

Comment:
combine the error wrapping into the return.

Reply:
fixed